### PR TITLE
[hi] Fix title mismatch in docs home for "Kubernetes Download"

### PR DESCRIPTION
**Pull request by pranav-pandey0804**
This PR fixes the issue #45456  where the Hindi localization home page had an incorrect title. 
Changed the title from **कुबेरनेट्स रिलीज नोट्स** to **कुबेरनेट्स डाउनलोड करें** to match the section's content about downloading Kubernetes.

Why This Change:

The old title suggested it was about release notes, which was misleading. The update ensures the title reflects the content accurately, making it easier for users to find the download information.

### CODE REVIEW

**Comment by pranav-pandey0804** [comment]
fixes issue #45456

File: content/hi/docs/home/_index.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ cards:
   button: प्रलेखन में योगदान करें 
   button_path: /docs/contribute
 - name: release-notes
-  title: कुबेरनेट्स रिलीज नोट्स 
+  title: कुबेरनेट्स डाउनलोड करें
   description: यदि आप कुबेरनेट्स इंस्टॉल कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें।
   button: कुबेरनेट्स डाउनलोड करें
   button_path: "/docs/setup/release/notes"
```
Comment (dipesh-rawat) [inline content/hi/docs/home/_index.md:61]: Considering the title change, should the `button_path` be also changed to align with this change?

**Comment by dipesh-rawat** [comment]
/retitle [hi] Fix title mismatch in docs home for "Kubernetes Download"

**Comment by dipesh-rawat** [comment]
> fixes issue #45456

Kindly incorporate the issue number into the PR description ([here](https://github.com/kubernetes/website/pull/45457#issue-2173389277)). This will help reviewers understand the context and assess the changes effectively.

**Comment by Shubhdeep02** [comment]
Is the issue still open?

File: content/hi/docs/home/_index.md (hunk 1 above)
Comment (pranav-pandey0804) [reply content/hi/docs/home/_index.md:61]: thanks for pointing that out, i'll update the button path too !

**Comment by pranav-pandey0804** [comment]
> > fixes issue #45456
> 
> Kindly incorporate the issue number into the PR description ([here](https://github.com/kubernetes/website/pull/45457#issue-2173389277)). This will help reviewers understand the context and assess the changes effectively.

@dipesh-rawat sure, thanks for suggestion!

**Comment by pranav-pandey0804** [comment]
> Is the issue still open?

Hi @Shubhdeep02 , thanks for the interest but I guess this PR will solve the issue,
any other help is welcomed!

File: content/hi/docs/home/_index.md (hunk 1 above)
Comment (pranav-pandey0804) [reply content/hi/docs/home/_index.md]: hi @dipesh-rawat , I've updated `button_path` but the download page is still in English language.

File: content/hi/docs/home/_index.md [hunk 2]
```diff
@@ -55,10 +55,10 @@ cards:
   button: प्रलेखन में योगदान करें 
   button_path: /docs/contribute
 - name: release-notes
-  title: कुबेरनेट्स रिलीज नोट्स 
+  title: कुबेरनेट्स डाउनलोड करें
   description: यदि आप कुबेरनेट्स इंस्टॉल कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें।
   button: कुबेरनेट्स डाउनलोड करें
-  button_path: "/docs/setup/release/notes"
+  button_path: "https://kubernetes.io/releases/download/"
```
Comment (dipesh-rawat) [inline content/hi/docs/home/_index.md:61]: We use site-relative links where feasible.
```suggestion
  button_path: "/releases/download"
```

File: content/hi/docs/home/_index.md (hunk 1 above)
Comment (dipesh-rawat) [reply content/hi/docs/home/_index.md]: In that case, the specific page could benefit from localization in Hindi. Once that's done, we can revisit this page and add 'hi' to the link.

File: content/hi/docs/home/_index.md (hunk 2 above)
Comment (pranav-pandey0804) [reply content/hi/docs/home/_index.md:61]: got that, thanks!

**Review by divya-mohan0209 — COMMENTED** [review]
Thanks for this fix!
/approve

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/lgtm

**Pull request merged**